packages/gcp/data_stream/loadbalancing_logs/fields/base-fields.yml

```
@@ -0,0 +1,20 @@
+- name: data_stream.type
+  type: constant_keyword
+  description: Data stream type.
+- name: data_stream.dataset
+  type: constant_keyword
+  description: Data stream dataset.
+- name: data_stream.namespace
+  type: constant_keyword
+  description: Data stream namespace.
+- name: '@timestamp'
+  type: date
+  description: Event timestamp.
+- name: event.module
+  type: constant_keyword
+  description: Event module
+  value: gcp
+- name: event.dataset
+  type: constant_keyword
+  description: Event dataset
+  value: gcp.load_balancer
```

I think this field must be equal to the datastream name to avoid rejection at ingestion time. `loadbalancing_metrics` had a similar https://github.com/elastic/integrations/pull/3986/commits/2c35e672b6b7c11ff8afdb2879a2e93a9f940915

```suggestion
  value: gcp.loadbalancing_logs
```